src/domain/prices/entities/asset-price.entity.ts

```diff
@@ -1,3 +1,3 @@
 export interface AssetPrice {
-  [assetName: string]: Record<string, number>;
+  [assetName: string]: Record<string, number | null>;
```

Comment:
This (and the following) type changes are changed according to the logic of `coingecko-api.service.ts`.